lib/usb-connection.js

```diff
@@ -22,14 +22,19 @@ function debugCommands(message) {
   log.debug(`(commands:usb) ${message}`);
 }
 
-var haveusb = true;
+var isUSBAvailable = true;
 try {
   var usb = require('usb');
   var VENDOR_REQ_OUT = usb.LIBUSB_REQUEST_TYPE_VENDOR | usb.LIBUSB_RECIPIENT_DEVICE | usb.LIBUSB_ENDPOINT_OUT;
   // var VENDOR_REQ_IN  = usb.LIBUSB_REQUEST_TYPE_VENDOR | usb.LIBUSB_RECIPIENT_DEVICE | usb.LIBUSB_ENDPOINT_IN;
-} catch (e) {
-  haveusb = false;
-  log.error('WARNING: No USB controller found on this system. Please run npm install -g t2-cli to compile USB drivers for your version of node');
+} catch (error) {
+  isUSBAvailable = false;
+
+  // do not exit the process during tests because usb is not needed to run them
+  if (!global.IS_TEST_ENV) {
+    log.error('Node version mismatch for USB drivers. Please run "npm rebuild --update-binary usb" to rebuild the USB drivers for your version of node');
```

Comment:
Let's take this opportunity to improve this for our users. Something like this: 

```js
const tags = require('common-tags');
```

```js
log.error('Node version mismatch for USB drivers.');
log.info(tags.stripIndent `
  To correct this issue, run the following command: 

  npm rebuild --update-binary usb

  This will rebuild the USB drivers for your version of Node.js
`);
```